.gitignore

```diff
@@ -1,6 +1,45 @@
 .DS_Store
+
+# Files generated by invoking Julia with --code-coverage
+*.jl.cov
+*.jl.*.cov
+
+# Files generated by invoking Julia with --track-allocation
+*.jl.mem
+
+# System-specific files and directories generated by the BinaryProvider and BinDeps packages
+# They contain absolute paths specific to the host computer, and so should not be committed
+deps/deps.jl
+deps/build.log
+deps/downloads/
+deps/usr/
+deps/src/
+
+# Build artifacts for creating documentation generated by the Documenter package
+docs/build/
+docs/site/
+
+# PythonCall added .CondaPkg to /docs
+docs/.CondaPkg
+
+# File generated by Pkg, the package manager, based on a corresponding Project.toml
+# It records a fixed state of all packages used by the project. As such, it should not be
+# committed for packages, but should be committed for applications that require a static
+# environment.
+
+# no jupyter notebook checkpoints
+.ipynb_*
+
+# no model output folders and data
+run_*/
 *.jld
 *.nc
+
+# no video outputs
+*.mp4
+
+# no nohup outputs
 nohup.out
-progress.txt
-parameter.txt
+
+# ignoring a folder on my computer (SW)
+/my_scripts
```

Comment:
This should probably be reverted for general users

Reply:
I agree, but I admittedly don't know how to push my code without having this line in the .gitignore (if I don't want that folder online). Do you usually manually remove it on Github?

Reply:
You can
- revert changes, push, change locally again, don't push again
- press `.` here, that opens a VS code session for this branch in which you can change things and push them without local changes